test/integration/test_kraken.py

```diff
@@ -0,0 +1,71 @@
+# Integration tests for kraken
+
+from builtins import super
+import os.path
+from os.path import join
+import tempfile
+import unittest
+import util.file
+import tools.kraken
+import tools.picard
+from test import TestCaseWithTmp
+from test.integration.snake import SnakemakeRunner
+
+
+class TestKrakenTiny(TestCaseWithTmp):
+
+    def setUp(self):
+        super().setUp()
+        self.kraken = tools.kraken.Kraken()
+        self.kraken.install()
+
+    @property
+    def data_dir(self):
+        return join(util.file.get_test_input_path(), 'TestToolKraken')
+
+    @property
+    def db_dir(self):
+        return os.path.join(self.data_dir, 'db')
+
+    def build_kraken_db(self):
+        db = os.path.join(tempfile.tempdir, 'db')
+        os.mkdir(db)
+        for d in ['library', 'taxonomy']:
+            realpath = os.path.join(self.db_dir, d)
+            name = os.path.join(db, d)
+            os.symlink(realpath, name)
+
+        # Minimizer len corresponds to memory/disk usage of index.
+        self.assertEqual(0, self.kraken.build(db, options={
+            '--minimizer-len': 10,
+            '--build': None,
+        }).returncode)
+        return db
+
+    def test_pipes(self):
```

Comment:
Please add the decorator:
@unittest.skipIf(sys.version_info < (3, 2), "python version is too old for snakemake")